db/schema.rb

```diff
@@ -54,8 +54,8 @@
     t.string   "experience"
     t.string   "interest"
     t.string   "school_id"
-    t.datetime "created_at",                                   :null => false
-    t.datetime "updated_at",                                   :null => false
+    t.datetime "created_at"
+    t.datetime "updated_at"
```

Comment:
I did a rebuild of the `db/schema.rb` and it kept out `null: false`, so must not be needed.